scripts/morty/analysis.py

```diff
@@ -0,0 +1,273 @@
+import solvers
+import numpy as np
+from data import DataHandler
```

Comment:
`data` is an extremely generic name. `data` could suggest a file that _contains_ data. I would recommend `data_handler` or something similar.